### PR TITLE
Merge pending pull requests

### PR DESCRIPTION
**Pull request by stuartpb**
Closes #4, closes #5, closes #6, closes #7, closes #8, closes #9, closes #10

### CODE REVIEW

File: githubc9.user.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+var target = document.getElementsByClassName("clone-options")[0];
+
+var link = document.createElement("a");
+link.className = "btn btn-sm sidebar-button c9-button";
+link.title = "Open this repository in Cloud9";
+link.rel = "nofollow";
+link.target = "_blank";
+
+var user = document.querySelector(".author>a>span").textContent;
```
Comment (nightwing) [inline githubc9.user.js]: this throws error on pages like https://github.com/, where .author is not present

File: githubc9.user.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+var repo = document.getElementsByClassName(
```
Comment (nightwing) [inline githubc9.user.js]: not important, but would be nicer to use `querySelector` here, for consistency.

File: githubc9.user.js (hunk 1 above)
Comment (stuartpb) [inline githubc9.user.js]: I'll have a fix for this in a few minutes.

File: githubc9.user.js (hunk 2 above)
Comment (stuartpb) [inline githubc9.user.js]: A fair point, I'll change all the element retrievals to use querySelector. (I was thinking it's simpler to use gEBCN, but that has undesired behavior if no such element exists.)

**Comment by stuartpb** [comment]
Wait, not yet...

File: githubc9.user.js (hunk 2 above)
Comment (stuartpb) [inline githubc9.user.js]: Actually, *is* it better to use querySelector over getElementsByClassName()[0]? Let me test that first.

**Comment by stuartpb** [comment]
Actually, let's merge this, and then discuss getElementsByClassName vs querySelector in a separate PR afterward. I'm getting kind of tired of running this 20-line Bash command every time I make a change:

```
git reset $(git commit-tree $(git write-tree) \
  -p $(git show-ref -s --heads master) \
  -p $(git show-ref -s --heads new-class) \
  -p $(git show-ref -s --heads remove-jq-ref) \
  -p $(git show-ref -s --heads remove-old-layout) \
  -p $(git show-ref -s --heads simpler-queries) \
  -p $(git show-ref -s --heads ssh-url) \
  -p $(git show-ref -s --heads style-in-code) \
  -p $(git show-ref -s --heads userscript) \
  -m "\
Merge branches 'new-class', 'remove-jq-ref', 'remove-old-layout', 'simpler-queries', 'ssh-url', 'style-in-code' and 'userscript' into roundup

Closes #4, closes #5, closes #6, closes #7, closes #8, closes #9, closes #10

        Conflicts:
                Makefile
                githubc9.user.js
                manifest.json
")
```

**Comment by nightwing** [comment]
Merged, thank you for great work!

> and then discuss getElementsByClassName vs querySelector 

I only meant that querySelector is shorter and looks cleaner than getElementsByClassName, but it's not very important.

**Pull request merged**